cloud-formation/membership-app.cf.yaml

```diff
@@ -93,6 +93,7 @@ Resources:
         Ref: ImageId
       SecurityGroups:
       - Ref: InstanceSecurityGroup
+      - Ref: WazuhSecurityGroup
```

Comment:
weirdly even though you put them that way, they appear the other way in the instance list. not a major issue but I did notice when checking our reservations
![image](https://user-images.githubusercontent.com/7304387/110346684-b3968c80-8027-11eb-800b-c050a8a26ebe.png)

Reply:
Ah, that's weird, shall I flip them then?

Reply:
no it's not an issue, I just noticed in the list because of that